develop.html.mako

```diff
@@ -91,135 +91,162 @@ seeking or providing support.</em></p>
-degree of control and collaborative ability. The pull request will be closed
-with a link to the review in Gerrit. Pull requests for code changes are never
-merged directly.
-
+    Pull requests are submitted to the GitHub repository at <a href="https://github
+    .com/sqlalchemy/sqlalchemy">https://github.com/sqlalchemy/sqlalchemy</a>. Once
+    accepted for review, assuming the pull request is altering the code itself and
+    not just correcting documentation issues, code review occurs within our Gerrit
+    system at <a
+    href="https://gerrit.sqlalchemy.org">https://gerrit.sqlalchemy.org</a> where we
+    can review, modify, and run it through continuous integration tests with a high
+    degree of control and collaborative ability. The pull request will be closed
+    with a link to the review in Gerrit. Pull requests for code changes are never
+    merged directly.
 </p>
 
-<p>Please note the <b>Guidelines for Pull Requests</b> which modify
-source code (e.g. not just documentation):
+<p>
+    Please note the <b>Guidelines for Pull Requests</b> which modify
+    source code (e.g. not just documentation):
+</p>
 
 <ul>
-<li><b>It is preferred to submit an issue in the issue tracker before submitting a
-pull request, </b> particularly for major changes.
-It is a frequent occurence that pull requests submitted either don't describe
-what problem they intend to solve, or they solve the problem in an inappropriate
-or sub-optimal way.   Having a bug or feature issue opened ahead of time
-will allow us to to understand the use case and discuss the best approach
-before the contributor makes the effort to write the code and tests.</li>
-
-<li><b>Always include tests with the change in code</b> - SQLAlchemy does not
-commit changes to code without a corresponding test - every line of code that
-isn't tested is technically a bug in itself.  When a pull request is submitted
-without tests, it <b>cannot be merged</b> until a test is written.  It is
-extremely common that we receive a pull request that is one line long,
-yet we need to write many dozens of lines to test it, since <b>99% of the work
-for any code change is writing the tests</b>.  <b>Please don't submit
-one-liner PRs without tests.  Only submit a <a href="participate.html#bugs">complete
-bug report</a> for code changes where you aren't able to provide tests</b>.</li>
-
-<li><b>Please follow up for requests for changes on the pull request, as well
-as subscribe to the Gerrit review to receive comments.</b> - Pull requests where
-requested changes are not responded towards will be closed.</li>
-
-<li>Please make sure you are configured to run the current <a href="#environment">Development Environment</a>
-    requirements, which will help ensure your PRs pass automatic tests.</li>
-
+    <li><a name="pr-requires-issue"></a>
+        <b>Issues for Pull Requests:
+        <em>It is required to submit an issue in the issue tracker before submitting a pull request.</em>
```

Comment:
@zzzeek This line was changed from "preferred" to "required". This is why I originally made the lower portion with  "preferred to make a PR"